ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Nagios_Plugins/tasks/nagios_Debian.yml

```diff
@@ -0,0 +1,35 @@
+---
+#################################################################################################
+# AdoptOpenJDK - Ansible Playbook to install Nagios plugins on Debian on x86 hardware #
+#################################################################################################
+
+###########################################################################
+# License Information:                                                    #
+# Nagios core and its plugins are lincesed under GPL                      #
+# For more information please see: https://www.nagios.com/legal/licenses/ #
+###########################################################################
+
+########################################
+# Install Nagios dependencies packages #
+########################################
+- name: Install Nagios plugins
+  apt: pkg={{ item }} state=latest update_cache=yes
+  with_items:
+    - fping
+    - gcc
+    - nagios-plugins
+    - nagios-plugins-common
+    - perl
+    - qstat
```

Comment:
The efficient variant is

```
- name: Install a list of packages
  apt:
    pkg:
    - foo
    - foo-tools
```

Reply:
Why gcc?

Reply:
> Why gcc?
I just copied it from `nagios_Ubuntu.yml`, so it was already in there- I'll remove it from both :-)